libs/constants/src/lib/health-care-services.ts

```diff
@@ -1,9 +1,6 @@
-export const HEALTH_CARE_SERVICES = {
-  0: 'כללית',
-  1: 'מאוחדת',
-  2: 'מכבי',
-  3: 'לאומית',
+export enum HEALTH_CARE_SERVICE {
```

Comment:
Such changes require verifying the report app to work, and also to verify data correction when working against the live data (where healthcare services are still indicated with "0", "1", etc.).